### PR TITLE
Publicity UI

### PR DESCRIPTION
**Pull request by Arushi1912**
Not final

### CODE REVIEW

**Comment by bhawesh96** [comment]
@Arushi1912 Things look great, just a few following changes required - 
1) It took long after clicking the SUBMIT button and there was nothing to show that its processing. Eventually, I ended up clicking it multiple times
2) "Upload your posters" box, when hovered should show the DISABLED cursor when Poster is not selected
3) The ripple effect of CheckBox is not well oriented
4) Finished container should be as of Book Event
5) Publicity Requests empty state, loading is not how My Events is
6) Remove un-required `console.log` statements

**Comment by Arushi1912** [comment]
Took care of all the mentioned points. Please check

**Comment by bhawesh96** [comment]
Squash your commits into one. You can use rebase. We need to have just one commit. Also, for commit message, refer my last commit on master

**Comment by bhawesh96** [comment]
The alignment got messed up I think
screenshot from 2018-12-26 22-03-40

**Comment by bhawesh96** [comment]
Code isn't standardised. Run standard to spot errors and fix them

**Comment by Arushi1912** [comment]
Worked with standardise. Please check

File: src/components/ClassroomComponent/PublicityPermComponent/FinishedContainer.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  render () {
+    return (
+      <div style={{ display: 'flex', flexDirection: 'column', alignItems: 'center', justifyContent: 'center', marginTop: this.props.isMobile ? '15%' : '2%', textAlign: 'center', marginBottom: 80, maxWidth: '98%' }}>
+        <div style={{ minHeight: 300 }}>
+          <img src={require('../../../assets/rocket-loader.gif')} style={{ width: this.props.isMobile ? '98%' : '50%' }} />
+        </div>
+
+        <p>{this.props.event && this.props.event.booker_fields.booker_name}, your publicity request titled "<span style={{ fontWeight: 700 }}>{this.props.event && this.props.event.title}</span>" has been requested for approval.</p>
+        <p>We'll notify you via email ({this.props.event && this.props.event.booker_fields.booker_email}) and SMS ({this.props.event && this.props.event.booker_fields.booker_contact})</p>
+        <p>Check live status under <Link to='dashboard/myEvents'>My Events</Link></p>
```
[the remark this answers is not in the document]
Comment (bhawesh96) [reply src/components/ClassroomComponent/PublicityPermComponent/FinishedContainer.js]: @Arushi1912

File: src/components/ClassroomComponent/PublicityPermComponent/Publicity.js [hunk 2]
```diff
@@ -1,43 +1,34 @@
...
       shouldCheck: false,
-      stepIndex:2,
+      stepIndex: 0,
       finished: false,
-      isFormValid:false,
-      booker_fields:  {
+      isFormValid: false,
+      bookerFields: {
         booker_name: '',
         booker_email: 'random@email.com',
```
Comment (bhawesh96) [inline src/components/ClassroomComponent/PublicityPermComponent/Publicity.js]: Remove these @Arushi1912

**Pull request closed without merging**